### PR TITLE
Projects Overview in Data Science Projects

### LINKED ISSUE #74: Move Project Overview doc under to-be section `Data Science Projects`
[Project Overview doc](https://operate-first.github.io/projects-overview) is a little out of place currently.

Move Project Overview doc under the `Data Science Projects` (currently named “Advanced Topics”) section since all project blogs are to be situated there.

This Project Overview doc can have links to all the other blogs later as we move all projects to the site.

### PR DESCRIPTION
**Pull request by aakankshaduggal**
This PR moves the Projects Overview document to the section Data Science Projects. 
Fixes #74

### CODE REVIEW

File: content/toc.yaml [hunk 1]
```diff
@@ -7,6 +7,9 @@
 - id: data-science-projects
   label: Data Science Projects
   links:
+    - id: projects-overview
+      label: Projects Overview
+      href: /projects-overview
```
Comment (oindrillac) [inline content/toc.yaml]: can you try changing this to `href: /data-science-projects/projects-overview`

**Review by oindrillac — CHANGES_REQUESTED** [review]
@aakankshaduggal I get an error as i try to test this.
Please see comment.

File: content/toc.yaml (hunk 1 above)
Comment (aakankshaduggal) [reply content/toc.yaml]: @oindrillac I have made the changes : https://aakankshaduggal.github.io/operate-first.github.io/data-science-projects/projects-overview

**Review by oindrillac — APPROVED** [review]
:ship: 
can you squash commits @aakankshaduggal

**Comment by oindrillac** [comment]
nvm @aakankshaduggal. sesheta handled it for us. :slightly_smiling_face:

**Pull request merged**